### PR TITLE
This Month in Our Test Infra (Nov 2024 issue)

### PR DESCRIPTION
**Pull request by jieyouxu**
Date range: 2024-11-05 to 2024-12-03

r? @Mark-Simulacrum (or @onur-ozkan)

[Rendered](https://github.com/jieyouxu/blog.rust-lang.org/blob/tmioti-nov-2024/posts/inside-rust/2024-12-09-test-infra-nov-2024.md)

### CODE REVIEW

**Review by Mark-Simulacrum — APPROVED** [review]

**Comment by jieyouxu** [comment]
boop @Mark-Simulacrum

**Comment by jieyouxu** [comment]
Rebased to today's date, no functional changes

File: posts/inside-rust/2024-12-09-test-infra-nov-2024.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+```rs
+// tests/ui/foo/auxiliary/my-proc-macro.rs
+```
+
+Thanks Eric!
+
+### rustc: make `rustc` consider itself a stable compiler when `RUSTC_BOOTSTRAP=-1` is set
+
+In [#132993](https://github.com/rust-lang/rust/pull/132993), I modified
```
Comment (weihanglo) [inline posts/inside-rust/2024-12-09-test-infra-nov-2024.md:67]: Just a drive-by: Maybe using @jieyouxu instead of `I`?

* Make the post in a more neutral tone.
* Can immediately know who worked on this when reading

Comment (jieyouxu) [reply posts/inside-rust/2024-12-09-test-infra-nov-2024.md:67]: I mean the post author is right there so lol

Comment (weihanglo) [reply posts/inside-rust/2024-12-09-test-infra-nov-2024.md:67]: I feel like calling yourself out gives the right amount of credits to your work. People just likely pay less attention to post authors 😞

It's up to you anyway. I love the post. Thank you :)
